src/d3d9/d3d9_interface.cpp

```diff
@@ -98,6 +98,11 @@ namespace dxvk {
       return S_OK;
     }
 
+    if (riid == __uuidof(ID3D9VkInteropInterface1)) {
```

Comment:
nit: Since this returns the same object as `ID3D9VkInteropInterface`, this should be folded into the if statement above (see the `IUnknown`/`IDirect3D9` thing further up, or any other instance of versioned COM interfaces).

Reply:
Done.